data/json/items/comestibles/junkfood.json

```diff
@@ -80,7 +80,7 @@
     "type": "COMESTIBLE",
     "id": "homemade_toasterpastry",
     "name": { "str": "toaster pastry", "str_pl": "toaster pastries" },
-    "copy-from": "toasterpastryfrozen",
+    "copy-from": "toastem4",
```

Comment:
same, Why change the copy from?

Reply:
homemade_toasterpastry is made by cooking a toastem4, not by cooking a toasterpastryfrozen

Reply:
reverted